src/test/java/com/exasol/rls/administration/scripts/AddUserToGroupIT.java

```diff
@@ -22,10 +23,10 @@
 
 @Testcontainers
 @Tag("integration")
+@Tag("slow")
 class AddUserToGroupIT extends AbstractAdminScriptIT {
     @Container
-    static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>(
-            EXASOL_DOCKER_IMAGE_REFERENCE).withReuse(true);
+    static final ExasolContainer<? extends ExasolContainer<?>> EXASOL = new ExasolContainer<>().withReuse(true);
```

Comment:
This is not using the matrix test docker-db version right?

Reply:
It is: https://github.com/exasol/exasol-testcontainers/blob/9b4f3c8e16bd12cd8d061e99e500faf4d59bcefd/src/main/java/com/exasol/containers/ExasolContainer.java#L82

Reply:
Well ok. I don't like the API since it is not intuitive. I would have preferred an approach like https://github.com/exasol/exasol-version-number-provider, where you do `ExasolContainer<>(versionNumberProvider.getVersionFromEnvVar());`.
Or at least: `new ExasolContainer<>().withVersionFromEnvVar()`. If you don't know this feature, it's just magic.

Anyway. Not an issue of this PR...